README.md

```diff
@@ -54,6 +54,8 @@ logdna register <email>
 # now paste the api key above
 ~~~~~
 
+If you have your LogDNA account in one of public cloud provider i.e. IBM Cloud. Please set the `ingester_domain` the default value for this option is https://logs.logdna.com. If you using LogDNA from IBM Cloud in `us-south` region please set the value of `ingester_domain`  to `https://logs.us-south.logging.cloud.ibm.com` 
```

Comment:
We're generally trying to keep provider-specific information out of the documentation, so I think rather than this note it would be better to add `ingester_domain` as an optional parameter in the `<match>` configuration section above. So it could be something like (@alfeng6 keep me honest): 

```
# ingester_domain https://logs.us-south.logging.cloud.ibm.com                    # which ingestion endpoint to send logs to, default https://logs.logdna.com
```

Reply:
@matt-march : Thanks for your comment and agreeing on to make these changes. I have updated the PR as per your comment.